tutorials/applications/Spin chain of 3 atoms in XY mode.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>Importing <code>Simulation</code>directly  from pulser will be deprecated soon. Add it as </p><p> </p><pre class="ql-syntax" spellcheck="false">from pulser.simulation import Simulation
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>